docs/pywbemclifeatures.rst

```diff
@@ -289,20 +289,44 @@ The following are examples of scalar property definitions:
 Interactively selecting INSTANCENAME
 ------------------------------------
 
-The INSTANCENAME argument has a certain complexity, particularly for
-associations and for classes with multiple keys.
+The INSTANCENAME argument can be complex, particularly for associations and
+classes with multiple keys.
+
+To simplify using INSTANCENAME on the command line, pywbemcli provides a wild
+card character "?" that can be used with commands in place of the INSTANCENAME
+keys on commands that have an INSTANCENAME argument. This allows the user to
+specify only the class name (as the INSTANCENAME argument value) with the wild
+card for the keys. Pywbemcli retrieves all instance names of that class from
+the server and presents the user with a select list from which an instance name
+can be chosen.
+
+Thus an INSTANCENAME argument on the command line can be specified in two ways:
+
+* By specifying an untyped WBEM URI of an instance path in the
+  ``INSTANCENAME`` argument. The namespace of the instance is the namespace
+  specified in the WBEM URI, or otherwise the namespace specified with the
+  ``-namespace``/``-n`` command option, or otherwise the default namespace
+  of the connection. Any host name in the WBEM URI will be ignored.
+  For details, see :ref:`Specifying the INSTANCENAME command argument`.
+
+* By specifying the ``--interactive`` command option and a class name in the
+  ``INSTANCENAME`` argument. The instances of the specified class are displayed
+  and the user is prompted for an index number to select an instance. The
+  namespace of the instance is the namespace specified with the
+  ``-namespace``/``-n`` command option, or otherwise the default namespace
+  of the connection.
+  For details, see :ref:`Interactively selecting INSTANCENAME`.
```

Comment:
The paragraphs above, from "Thus an INSTANCENAME" to here do not make sense in this section "Interactively selecting INSTANCENAME". They would make more sense (without the section references) when combining the two sections into one.

Reply:
Completely redid both the descriptions in pywbemclicommands and the featues section.  Made the section on that argument very short in each command spec and put it all together into a single section in features.  Lets see if that reads better.

Reply:
Reviewed and agreed.